code/game/objects/items/weapons/shields.dm

```diff
@@ -35,6 +35,19 @@
 	icon_state = "roman_shield"
 	item_state = "roman_shield"
 
+/obj/item/weapon/shield/riot/buckler
+	name = "wooden buckler"
+	desc = "A medieval wooden buckler."
+	icon_state = "buckler"
+	item_state = "buckler"
+	materials = list()
+
+/obj/item/weapon/shield/riot/buckler/IsShield()
+	if(prob(60))
+		return 1
+	else
+		return 0
+

```

Comment:
Yeah--I don't personally like it, but it does give a rough ~30% chance for it to block something or not (granted that's with the assumption of 0 damage).

Later one, if/when we refactor hit checks this can go away.

Maybe it should be renamed to Schrodinger's shield.